airflow/contrib/operators/mlengine_operator.py

```diff
@@ -151,6 +151,9 @@ class MLEngineBatchPredictionOperator(BaseOperator):
         have doamin-wide delegation enabled.
     :type delegate_to: str
 
```

Comment:
Trim this newline please

Reply:
ok but, I tried to keep the same blank line as in other parameters